js/classes/entities/midground/particle_effect.js

```diff
@@ -232,4 +352,89 @@ class ParticleEffect {
             behavior: ParticleEffect.RISE
         };
     }
+
+    static get GOLD_SPARKLE() {
+        return {
+            spread: 80,
+            size: 4,
+            amount: 5,
+            lifetime: 0.3,
+            color: COLORS.GOLD,
+            opacity: 0.7,
+            behavior: ParticleEffect.SLOW_EXPAND
+        };
+    }
+
+    static get RED_SPARKLE() {
+        return {
+            spread: 80,
+            size: 4,
+            amount: 5,
+            lifetime: 0.3,
+            color: COLORS.RED,
+            opacity: 0.7,
+            behavior: ParticleEffect.SLOW_EXPAND
+        };
+    }
+
+    static get GREEN_SPARKLE() {
+        return {
+            spread: 80,
+            size: 4,
+            amount: 5,
+            lifetime: 0.3,
+            color: COLORS.GREEN,
+            opacity: 0.7,
+            behavior: ParticleEffect.SLOW_EXPAND
+        };
+    }
+
+    static get GRAY_SPARKLE() {
```

Comment:
Can we JSdoc all of these or is there some standard I am not aware of that says we shouldn't?